### PR TITLE
Prettier eslint

### PR DESCRIPTION
**Pull request by 0xEG**
- create new git project by npm;
- bootstrap it with eslint and prettier;
- add and set up npm script to run eslint (.eslintrc.json);
- add and set up npm script to run prettier (.prettierrc.json );
- add .gitignore file;

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,8 @@
+node_modules
+/log/
+*.log
+.DS_Store
+.eslintrc.js
+
+package-lock.json
+package.json
```
Comment (JekRock) [inline .gitignore]: These files should not be excluded.
Basically, this is the first point in the issue: `create a new npm project`.
Please read the following to get to know more about `package.json` and `package-lock.json`
- https://nodejs.dev/learn/an-introduction-to-the-npm-package-manager
- https://nodejs.dev/learn/the-package-json-guide
- https://nodejs.dev/learn/the-package-lock-json-file

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,8 @@
+node_modules
+/log/
+*.log
+.DS_Store
+.eslintrc.js
```
Comment (JekRock) [inline .gitignore]: Why is the eslint configuration excluded?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+# 
+Create a backend for the RealWorld app.
```
Comment (JekRock) [inline README.md]: It seems like these lines should be a single like to make a heading

File: README.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
...
+- create new project and bootstrap it with Prettier and Eslint
+- make HTTP server on Fastify;
+- testing server by Jest;
```
Comment (JekRock) [inline README.md]: Do we have these two points in this PR? If no, please remove them for now

**Review by JekRock — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (0xEG) [reply .gitignore]: Tks for review. I`ll back it to head directory.

File: .gitignore (hunk 2 above)
Comment (0xEG) [reply .gitignore]: Fixed

File: README.md (hunk 3 above)
Comment (0xEG) [reply README.md]: Done

File: README.md (hunk 4 above)
Comment (0xEG) [reply README.md]: Done

**Review by JekRock — APPROVED** [review]

**Pull request merged**